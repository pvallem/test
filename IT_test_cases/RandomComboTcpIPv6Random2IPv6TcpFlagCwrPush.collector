bidir_flow_info {
  key {
    src_address: ".9\264\316w\013\374\223j\277\217s\363\036\223:"
    dst_address: "\267:\226\004\254\212\255\327\245\214\341\244E%\214\265"
    src_port: 35393
    dst_port: 63307
    proto: 6
    tenant_id: 10
    flow_start_time: 1388563200000020
    key_type: IPV6
  }
  flow_info {
    start_time: 1388563200000020
    ip_info {
      last_cached_ttl: 8
      qos: 0
    }
    tcp_info {
      sequence_num: 10
      null_flags: 1
      syn_fin_rst_ack_all_zero: 1
      option_len_bins {
      }
    }
    flow_features {
      packet_count: 1
    }
    end_time: 1388563200000020
    sensor_id: "127.0.0.1:5675"
  }
}
bidir_flow_info {
  key {
    src_address: "\001\002\003\004\000\000\000\000\000\000\000\000\000\000\000\000"
    dst_address: "\005\006\007\010\000\000\000\000\000\000\000\000\000\000\000\000"
    src_port: 111
    dst_port: 222
    proto: 6
    tenant_id: 10
    flow_start_time: 1388563200000010
    key_type: IPV6
  }
  flow_info {
    start_time: 1388563200000010
    ip_info {
      qos: 0
    }
    flow_features {
      packet_count: 2
    }
    end_time: 1388563200000030
    sensor_id: "127.0.0.1:5675"
  }
}
bidir_flow_info {
  key {
    src_address: "\226\331\002a\244F\250\364\236\223\245T_\312\234\317"
    dst_address: "\365O\276\365\t\240`\013\203\017@Bl\200P`"
    src_port: 47325
    dst_port: 31748
    proto: 6
    tenant_id: 10
    flow_start_time: 1388563200000040
    key_type: IPV6
  }
  reverse_flow_info {
    start_time: 1388563200000040
    ip_info {
      last_cached_ttl: 8
      qos: 0
    }
    tcp_info {
      sequence_num: 20
      null_flags: 1
      syn_fin_rst_ack_all_zero: 1
      option_len_bins {
      }
    }
    flow_features {
      packet_count: 1
    }
    end_time: 1388563200000040
    sensor_id: "127.0.0.1:5675"
  }
}
bidir_flow_info {
  key {
    src_address: "\303{\323\016\r\035\310\307\020\377{\362)\274I\324"
    dst_address: "\341p0\010.(\365I\316I\255\217u>\261\344"
    src_port: 43297
    dst_port: 53800
    proto: 6
    tenant_id: 10
    flow_start_time: 1388563200000000
    key_type: IPV6
  }
  reverse_flow_info {
    start_time: 1388563200000000
    ip_info {
      last_cached_ttl: 8
      qos: 0
    }
    tcp_info {
      sequence_num: 0
      null_flags: 1
      syn_fin_rst_ack_all_zero: 1
      option_len_bins {
      }
    }
    flow_features {
      packet_count: 1
    }
    end_time: 1388563200000000
    sensor_id: "127.0.0.1:5675"
  }
}
