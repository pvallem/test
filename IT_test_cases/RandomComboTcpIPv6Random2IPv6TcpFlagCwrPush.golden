flow_info {
  key {
    src_address: "\341p0\010.(\365I\316I\255\217u>\261\344"
    dst_address: "\303{\323\016\r\035\310\307\020\377{\362)\274I\324"
    src_port: 53800
    dst_port: 43297
    proto: 6
    tenant_id: 10
    flow_start_time: 1388563200000000
    key_type: IPV6
  }
  start_time: 1388563200000000
  ip_info {
    last_cached_ttl: 8
    qos: 0
  }
  tcp_info {
    sequence_num: 0
    null_flags: 1
    syn_fin_rst_ack_all_zero: 1
    option_len_bins {
    }
  }
  flow_features {
    packet_count: 1
  }
  end_time: 1388563200000000
}
flow_info {
  key {
    src_address: "\001\002\003\004\000\000\000\000\000\000\000\000\000\000\000\000"
    dst_address: "\005\006\007\010\000\000\000\000\000\000\000\000\000\000\000\000"
    src_port: 111
    dst_port: 222
    proto: 6
    tenant_id: 10
    flow_start_time: 1388563200000010
    key_type: IPV6
  }
  start_time: 1388563200000010
  ip_info {
    qos: 0
  }
  flow_features {
    packet_count: 2
  }
  end_time: 1388563200000030
}
flow_info {
  key {
    src_address: ".9\264\316w\013\374\223j\277\217s\363\036\223:"
    dst_address: "\267:\226\004\254\212\255\327\245\214\341\244E%\214\265"
    src_port: 35393
    dst_port: 63307
    proto: 6
    tenant_id: 10
    flow_start_time: 1388563200000020
    key_type: IPV6
  }
  start_time: 1388563200000020
  ip_info {
    last_cached_ttl: 8
    qos: 0
  }
  tcp_info {
    sequence_num: 10
    null_flags: 1
    syn_fin_rst_ack_all_zero: 1
    option_len_bins {
    }
  }
  flow_features {
    packet_count: 1
  }
  end_time: 1388563200000020
}
flow_info {
  key {
    src_address: "\365O\276\365\t\240`\013\203\017@Bl\200P`"
    dst_address: "\226\331\002a\244F\250\364\236\223\245T_\312\234\317"
    src_port: 31748
    dst_port: 47325
    proto: 6
    tenant_id: 10
    flow_start_time: 1388563200000040
    key_type: IPV6
  }
  start_time: 1388563200000040
  ip_info {
    last_cached_ttl: 8
    qos: 0
  }
  tcp_info {
    sequence_num: 20
    null_flags: 1
    syn_fin_rst_ack_all_zero: 1
    option_len_bins {
    }
  }
  flow_features {
    packet_count: 1
  }
  end_time: 1388563200000040
}
